package.json

```diff
@@ -64,5 +65,10 @@
     "test": "npm run lint && npm run fast-test",
     "view-cover": "npm run cover -- --report=html && opn ./coverage/index.html"
   },
-  "private": false
+  "files": [
```

Comment:
Maybe it's just that I'm coming from Python, but it seems weird to specifically ignore something that just happens to be kicking around in the directory (like my `.idea` project dir, or coverage results.)